backend/src/main/java/capstone/backend/model/db/Product.java

```diff
@@ -17,25 +17,27 @@
 @AllArgsConstructor
 @Builder
 @Entity
+@With
```

Comment:
why you need this? the product is not final?

Reply:
mainly to make it easier to construct sample data for my tests